sonar-dotnet-shared-library/pom.xml

```diff
@@ -14,7 +14,7 @@
   <description>Share code between C# and VB.NET plugins</description>
 
   <properties>
-    <sonar.version>7.3</sonar.version>
+    <sonar.version>7.4.0.16659</sonar.version>
```

Comment:
Previously it looks like this version was just _major.minor_. Out of interest, is there a reason for moving to a four-part version now?

Reply:
This is a QA build, we'll move to an official release when available.